debian/control

```diff
@@ -26,7 +26,7 @@ Package: xpu-smi
 Architecture: amd64
 Depends:
  libze1 (>= 1.17.6),
- igsc (>=0.8.4),
+ intel-gsc (>=0.9.3),
```

Comment:
Does that mean that xpu-smi at the current version does not work with intel-gsc at version 0.8.4 ?

Reply:
We will not have a version 0.8.4, so I'm just updating to the version that exists

Reply:
We will not have a version 0.8.4, so I'm just updating to the version that exists